editor/editor_plugin.h

```diff
@@ -87,7 +87,7 @@ class EditorInterface : public Node {
 
 	EditorSelection *get_selection();
 	//EditorImportExport *get_import_export();
-	EditorSettings *get_editor_settings();
+	Ref<EditorSettings> get_editor_settings();
```

Comment:
Shouldn't the same be done for the surrounding getters here?

Reply:
No, those don't inherit Reference. I added a check to the C# bindings generator, so it will error out in the future if there are more cases like this.